### PR TITLE
Enterprise 1.10

### PR DESCRIPTION
**Pull request by lwandzura**
Closes:
https://github.com/influxdata/docs-v2/issues/4011
https://github.com/influxdata/docs-v2/issues/3230
https://github.com/influxdata/docs-v2/issues/3961
https://github.com/influxdata/docs-v2/issues/3914

New 1.10 release docs

@davidby-influx Please review the [API sections](https://github.com/influxdata/docs-v2/pull/4416/commits/1f3f1e0d7a3e2a2b47a6dd90993dfd491e1259be)

### CODE REVIEW

File: content/enterprise_influxdb/v1.10/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -0,0 +1,1165 @@
...
+  enterprise_influxdb_1_10_ref:
+    name: Release notes
+    weight: 10
+    parent: About the project
+---
+
+## 1.10 [2022-09-07]
+
+### Features
+- Add  /v2/api/buckets support for create, delete, list, retrieve, and update operations.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.10/about-the-project/release-notes-changelog.md:15]:
```suggestion
- Add [/api/v2/buckets](/enterprise_influxdb/v1.10/tools/api/#apiv2buckets-http-endpoint) support for create, delete, list, retrieve, and update operations.
```

File: content/enterprise_influxdb/v1.10/about-the-project/release-notes-changelog.md [hunk 2]
```diff
@@ -0,0 +1,1165 @@
...
+- Add  /v2/api/delete support.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.10/about-the-project/release-notes-changelog.md:16]:
```suggestion
- Add [/api/v2/delete](/enterprise_influxdb/v1.10/tools/api/#apiv2delete-http-endpoint) support.
```

**Review by kelseiv — CHANGES_REQUESTED** [review]

**Review by kelseiv — APPROVED** [review]

**Comment by kelseiv** [comment]
cc @veereshhosakere

**Review by davidby-influx — CHANGES_REQUESTED** [review]
Will continue to review more, but wanted to get the notes on HH retry-interval to you

File: content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md [hunk 3]
```diff
@@ -0,0 +1,1628 @@
...
+
+The time period after which the hinted handoff retries a write after the write fails.
+
+Environment variable: `INFLUXDB_HINTED_HANDOFF_RETRY_INTERVAL`
+
+#### `retry-max-interval`
+
+Default is `"10s"`.
+
+The maximum interval after which the hinted handoff retries a write after the write fails.
```
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md:747]: fixed

File: content/enterprise_influxdb/v1.10/tools/api.md [hunk 4]
```diff
@@ -154,6 +156,70 @@ curl -XGET "localhost:8086/health"
 
 ---
 
+### `/api/v2/buckets/` HTTP endpoint
```
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/tools/api.md:159]: @kelseiv not sure how to proceed here, we do have separate API docs where I do believe this is covered

File: content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md [hunk 5]
```diff
@@ -0,0 +1,1628 @@
...
+#### `retry-rate-limit`
+
+Default is `0`.
+
+The rate limit (in bytes per second) that hinted handoff retries hints. A value of `0` disables the rate limit.
+
+Environment variable: `INFLUXDB_HINTED_HANDOFF_RETRY_RATE_LIMIT`
+
+#### `retry-interval`
+
```
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md:736]: @davidby-influx the exponential back off was first mentioned in previous release notes (v1.9.3) but nothing else was added int he docs.  Is the default still 1s for retry-interval?  I'll include your wording under retry-interval

File: content/enterprise_influxdb/v1.10/tools/api.md [hunk 6]
```diff
@@ -154,6 +156,70 @@ curl -XGET "localhost:8086/health"
...
+  Accepts `ns` (nanoseconds), `us`(microseconds), `ms` (milliseconds) and `s` (seconds).
+
+**Include the following HTTP header:**
+
+- `Authorization`: In InfluxDB 2.x uses [API Tokens](/influxdb/v2.0/security/tokens/)
+  to access the platform and all its capabilities.
+  InfluxDB v1.x uses a username and password combination when accessing the HTTP APIs.
+  Use the Token schema to provide your InfluxDB 1.x username and password separated by a colon (`:`).
+  For example: `Authorization: Token username:password`.
+
```
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/tools/api.md:211]: @kelseiv I think we need to structure the API info for Enteprrise the same way we do for 2.x OSS (consistency!) - that would take some work, and we need to get this release out.  I can create a separate issue for that - thoughts?

File: content/enterprise_influxdb/v1.10/tools/api.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/enterprise_influxdb/v1.10/tools/api.md:159]: @lwandzura, we can add the following links to the Enterprise doc:

[/api/v2/buckets](/influxdb/latest/api/#tag/Buckets)
[create](/influxdb/latest/api/#operation/PostBuckets)
[delete](/influxdb/latest/api/#operation/DeleteBucketsID)
[list](/influxdb/latest/api/#operation/GetBuckets)
[update](/influxdb/latest/api/#operation/PatchBucketsID)
[retrieve](/influxdb/latest/api/#operation/GetBucketsID)


Currently, in Enterprise docs, we say 
_bucket: Provide the database name and retention policy separated by a forward slash (/). For example: database/retention-policy. Empty retention policies map to the default retention policy._
@davidby-influx do folks need to add colons before db and rp in the request?

cc @jstirnaman - if we don't already, may be helpful to add a note regarding the 1.x db/rp in api ref docs for Delete a bucket, Retrieve a bucket, and Update a bucket

File: content/enterprise_influxdb/v1.10/tools/api.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/enterprise_influxdb/v1.10/tools/api.md:211]: @lwandzura, yes, let's discuss

File: content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md [hunk 7]
```diff
@@ -736,17 +736,15 @@ Environment variable: `INFLUXDB_HINTED_HANDOFF_RETRY_RATE_LIMIT`
 
 Default is `"1s"`.
 
-The time period after which the hinted handoff retries a write after the write fails.
+The time period after which the hinted handoff retries a write after the write fails. There is an exponential back-off, which starts at 1 second and increases with each failure. It resets on success.
```
Comment (davidby-influx) [inline content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md:739]: "... increases with each failure until it reaches `retry-max-interval`.  Retrials will then occur at `retry-max-interval`.  On any successful retrial, the waiting period will be reset to `retry-interval`

File: content/enterprise_influxdb/v1.10/tools/api.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/tools/api.md:159]: added links

[the remark this answers is not in the document]
Comment (davidby-influx) [reply content/enterprise_influxdb/v1.10/tools/api.md:159]: @kelseiv - do not add colons.  Those are indicators to the Route code to extract those values from the expression....

[the remark this answers is not in the document]
Comment (davidby-influx) [reply content/enterprise_influxdb/v1.10/tools/api.md:159]: And a lot of the V2 bucket API (like owners and labels and calls dealing with them) are not supported in 1.X.

File: content/enterprise_influxdb/v1.10/tools/influx_inspect.md [hunk 8]
```diff
@@ -0,0 +1,587 @@
...
+The `influx_inspect` commands are summarized here, with links to detailed information on each of the commands.
+
+* [`buildtsi`](#buildtsi): Converts in-memory (TSM-based) shards to TSI.
+* [`deletetsm`](#deletetsm): Bulk deletes a measurement from a raw TSM file.
+* [`dumptsi`](#dumptsi): Dumps low-level details about TSI files.
+* [`dumptsm`](#dumptsm): Dumps low-level details about TSM files.
+* [`dumptsmwal`](#dumptsmwal): Dump all data from a WAL file.  
+* [`export`](#export): Exports raw data from a shard in InfluxDB line protocol format.
+* [`report`](#report): Displays a shard level report.
+* [`reporttsi`](#reporttsi): Reports on cardinality for measurements and shards.
```
Comment (davidby-influx) [inline content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: Do we document the `report-disk` command?

Comment (davidby-influx) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: @lesam wrote it.

**Review by davidby-influx — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.10/tools/api.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/enterprise_influxdb/v1.10/tools/api.md:159]: @kelseiv @lwandzura @sunbryely-influxdata We should add some request and maybe code samples for these in `openapi`.

File: content/enterprise_influxdb/v1.10/tools/influx_inspect.md (hunk 8 above)
Comment (lesam) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: We should have it documented - https://github.com/influxdata/docs-v2/issues/2112

File: content/enterprise_influxdb/v1.10/tools/api.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/enterprise_influxdb/v1.10/tools/api.md:159]: > do folks need to add colons before db and rp in the request?
@kelseiv I don't think so - it's part of the InfluxDB code; in the `Route` URLs, the `/:param` indicates a variable in the route pattern.

In docs, we'd say _something_ like this:

To use your database name and retention policy to update a bucket, pass the updated fields in the request body to the ` PATCH http://localhost:8086/api/v2/buckets/DATABASE_NAME/RETENTION_POLICY` InfluxDB `/api/v2` endpoint.

In the URL, replace the following:

- **`DATABASE_NAME`**: the name of your 1.x database mapped to the bucket
- **`RETENTION_POLICY`**: the name of your 1.x retention policy mapped to the bucket

File: content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md (hunk 7 above)
Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md:739]: added the new verbiage

File: content/enterprise_influxdb/v1.10/tools/influx_inspect.md (hunk 8 above)
Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: @davidby-influx weird - that's no longer there in the code, but there was a PR for it last year

Comment (davidby-influx) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: @lwandzura - what's no longer in the code?  

https://github.com/influxdata/influxdb/blob/1.10/cmd/influx_inspect/reportdisk/reportdisk.go

Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: it's not in our docs code
https://docs.influxdata.com/enterprise_influxdb/v1.9/tools/influx_inspect/

Comment (lwandzura) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: I'll add back the report-disk after the v1.10 release docs are published - issue https://github.com/influxdata/docs-v2/issues/4440

File: content/enterprise_influxdb/v1.10/tools/api.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/enterprise_influxdb/v1.10/tools/api.md:211]: for now, added a predicate example in this doc, and linked to delete api doc + delete  predicate syntax

File: content/enterprise_influxdb/v1.10/tools/influx_inspect.md (hunk 8 above)
Comment (kelseiv) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: @lwandzura thank you for creating this issue https://github.com/influxdata/docs-v2/issues/4440

Comment (kelseiv) [reply content/enterprise_influxdb/v1.10/tools/influx_inspect.md:39]: @lwandzura think we can push out this Enterprise 1.10 content and loop back around on report-disk

**Pull request merged**